docs/docs/inputs_outputs.md

```diff
@@ -7,40 +7,65 @@ This section covers what data you can pass to zamba and what the output looks li
 Using data collected and annotated by partners at [The Max Planck Institute for
 Evolutionary Anthropology](https://www.eva.mpg.de/index.html) and [Chimp &
 See](https://www.chimpandsee.org/), the prepackaged model in `zamba` was
-trained to detect 23 species of animal––as well as `blank` videos (no animal
+trained to detect 31 species of animal––as well as `blank` videos (no animal
 present).
 
 The possible class labels in `zamba`'s default model are:
 
+* `aardvark`
+* `antelope_duiker`
+* `badger`
+* `bat`
 * `bird`
 * `blank`
 * `cattle`
-* `chimpanzee`
-* `duiker`
+* `cheetah`
+* `chimpanzee_bonobo`
+* `civet_genet`
 * `elephant`
-* `forest buffalo`
+* `equid`
+* `forest_buffalo`
+* `fox`
+* `giraffe`
 * `gorilla`
+* `hare_rabbit`
 * `hippopotamus`
 * `hog`
 * `human`
 * `hyena`
-* `large ungulate`
+* `large_flightless_bird`
 * `leopard`
 * `lion`
-* `other (non-primate)`
-* `other (primate)`
+* `mongoose`
+* `monkey_prosimian`
 * `pangolin`
 * `porcupine`
 * `reptile`
 * `rodent`
-* `small antelope`
-* `small cat`
-* `wild dog`
+* `small_cat`
+* `wild_dog_jackal`
+
+<!-- TODO: add details of european model. who collected and labels the data?><!-->
+
+*New in Zamba v2:* Thanks to new data shared by [<SOURCE>], `zamba` now also includes a prepackaged model to identify 11 common European species. The class labels in `zamba`'s European model are:
+
+* `bird`
+* `blank`
+* `domestic_cat`
+* `european_badger`
+* `european_beaver`
+* `european_hare`
+* `european_roe_deer`
+* `north_american_raccoon`
+* `red_fox`
+* `unidentified`
+* `weasel`
+* `wild_boar`
 
 ## What Videos Can I use?
 
-The `zamba` models were trained using 15 second `.mp4` videos. Although videos
-longer than 15 seconds will process without error, we recommend segmenting your videos into 15 second segments for the most accurate results since videos are downsampled to a smaller number of frames during processing. This can be done with the `ffmpeg` commandline tool. Here's an example:
+The `zamba` models were trained using 15 second `.mp4` videos. <!--TODO: noemie's data for depth estimation included .avi videos, does the data used to train zamba still only include .mp4?><!--> Although videos
+longer than 15 seconds will process without error, we recommend segmenting your videos into 15 second segments for the most accurate results since videos are downsampled to a smaller number of frames during processing. This can be done with the `ffmpeg` commandline tool. <!-- TODO: is that still true? did we still only train on 15 second videos?><!--> Here's an example:
```

Comment:
Is this still true that it was trained on only 15 second .mp4 videos? The s3 directory has a lot of .avi videos and videos with different lengths

Reply:
nope, that was competition specific. these are now largely 1 minute videos